### ISSUE TITLE
[BUG] uninitialized constant SmartKv::Meat::Set (NameError)

### ISSUE THREAD

**Issue by noraj** [styd/apexcharts.rb#55]
**Describe the bug**

It seems there is an issue with the dependency `smart_kv`

```
$ erb charts.html.erb > charts.html
Traceback (most recent call last):
        7: from /home/noraj/.rbenv/versions/2.7.0/bin/erb:175:in `<main>'
        6: from /home/noraj/.rbenv/versions/2.7.0/bin/erb:154:in `run'
        5: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/2.7.0/erb.rb:890:in `run'
        4: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/2.7.0/erb.rb:905:in `result'
        3: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/2.7.0/erb.rb:905:in `eval'
        2: from charts.html.erb:2:in `<main>'
        1: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require'
/home/noraj/.rbenv/versions/2.7.0/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require': cannot load such file -- apexcharts (LoadError)
        22: from /home/noraj/.rbenv/versions/2.7.0/bin/erb:175:in `<main>'
        21: from /home/noraj/.rbenv/versions/2.7.0/bin/erb:154:in `run'
        20: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/2.7.0/erb.rb:890:in `run'
        19: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/2.7.0/erb.rb:905:in `result'
        18: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/2.7.0/erb.rb:905:in `eval'
        17: from charts.html.erb:2:in `<main>'
        16: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:156:in `require'
        15: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:168:in `rescue in require'
        14: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:168:in `require'
        13: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/apexcharts-0.1.8/lib/apexcharts.rb:3:in `<top (required)>'
        12: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:72:in `require'
        11: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:72:in `require'
        10: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/apexcharts-0.1.8/lib/apexcharts/helper.rb:6:in `<top (required)>'
         9: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/apexcharts-0.1.8/lib/apexcharts/helper.rb:6:in `require_relative'
         8: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/apexcharts-0.1.8/lib/apexcharts/options_builder.rb:4:in `<top (required)>'
         7: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/apexcharts-0.1.8/lib/apexcharts/options_builder.rb:4:in `require_relative'
         6: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/apexcharts-0.1.8/lib/apexcharts/options/root.rb:1:in `<top (required)>'
         5: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/apexcharts-0.1.8/lib/apexcharts/options/root.rb:1:in `require_relative'
         4: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/apexcharts-0.1.8/lib/apexcharts/options/div_attributes.rb:1:in `<top (required)>'
         3: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/apexcharts-0.1.8/lib/apexcharts/options/div_attributes.rb:2:in `<module:ApexCharts>'
         2: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/apexcharts-0.1.8/lib/apexcharts/options/div_attributes.rb:3:in `<class:DivAttributes>'
         1: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/smart_kv-0.2.7/lib/smart_kv/meat.rb:26:in `optional'
/home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/smart_kv-0.2.7/lib/smart_kv/meat.rb:38:in `init_optional': uninitialized constant SmartKv::Meat::Set (NameError)
```

So we could think about an upstream issue with SmartKv but the version used (0.2.7) is an exact version in the gemspec file nad it's the same version used since the first commit of apexcharts.rb.

```
$ irb
irb(main):001:0> require 'apexcharts'
=> true
irb(main):001:0> require 'smart_kv'
=> true
irb(main):002:0> SmartKv::Meat::Set
Traceback (most recent call last):
        4: from /home/noraj/.rbenv/versions/2.7.0/bin/irb:23:in `<main>'
        3: from /home/noraj/.rbenv/versions/2.7.0/bin/irb:23:in `load'
        2: from /home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/irb-1.2.1/exe/irb:11:in `<top (required)>'
        1: from (irb):2
NameError (uninitialized constant SmartKv::Meat::Set)

$ gem list | grep apexcharts
apexcharts (0.1.8)
```

**To Reproduce**

```
$ rbenv local 2.7.0
$ gem install apexcharts
$ rbenv rehash
$ erb charts.html.erb > charts.html
```

charts.html.erb from [here](https://github.com/styd/apexcharts.rb#plain-htmlerb-without-framework)

```erb
<script src="https://cdn.jsdelivr.net/npm/apexcharts"></script>
<% require 'apexcharts' %>
<% include ApexCharts::Helper %>
```

**Expected behavior**

Not crashing when generating erb.

**Desktop (please complete the following information):**
 - OS: ArchLinux
 - Browser Firefox
 - Version 0.1.8

**Comment by styd** [comment]
@noraj, can you try installing from the `master` branch and tell me if it solves your issue.

**Comment by noraj** [comment]
@styd Yes with your fix + the new SmartKv release it works now. Thank you very much.

**Comment by styd** [comment]
You're welcome. Glad I could help.
I've just released current `master` branch as version 0.1.9.
You can use this version next time.
